config/routes.rb

```diff
@@ -95,6 +95,9 @@ def matches?(request)
     resources :payroll_runs, only: [:index, :new, :create, :show] do
       resources :payment_confirmation_report_uploads, only: [:new, :create]
       resource :download, only: [:new, :create, :show], controller: "payroll_run_downloads"
+      resources :payments, only: [:destroy] do
+        get :delete, to: "payments#delete", on: :member
```

Comment:
We can rely on the router conventions to map to the controller and action without being explicit:

```suggestion
        get :delete, on: :member
```